Core/ContentBlockerRulesSource.swift

```diff
@@ -41,11 +41,16 @@ class DefaultContentBlockerRulesSource: ContentBlockerRulesSource {
     }
 
     var tempListEtag: String? {
-        return UserDefaultsETagStorage().etag(for: .temporaryUnprotectedSites)
+        return PrivacyConfigurationManager.shared.fetchedData?.etag
+            ?? PrivacyConfigurationManager.shared.embeddedData.etag
```

Comment:
I have one follow up question: how often can configuration change and how many of these changes can be unrelated to content blocker (temp list, etc)? Right now any change to that config will force every device to rebuild content blocking, so if we are going to have frequent updates that are unrelated to that, I'd rather see some other UID to be provided for that subset of data.

Reply:
I think the most common updates will be to the `unprotectedTemporary` or `contentBlocking` exemptions. Both would require a rebuild of the CBRs. Feature toggles won't change much and so far we haven't seen many instances of needing exception rules for the other features. I don't imagine it changing much more often than the old list did.

Reply:
Ok, so that's current state and it is fine. Now - outside of this PR - can you think of steps to guarantee that any changes to that (e.g. introducing new processes around its release or adding new functionalities) will take into account possible impact of any updates?

Reply:
Remote Config will likely become an AoR. I'll make sure these considerations are documented and considered as things change.